### PR TITLE
Updated makeflow options to include GC

### PR DESCRIPTION
**Pull request by nhazekam**
Addressed #1670  reintroduces garbage collection options to the help message.

### CODE REVIEW

File: makeflow/src/makeflow.c [hunk 1]
```diff
@@ -1405,7 +1408,6 @@ int main(int argc, char *argv[])
 		{"gc", required_argument, 0, 'g'},
 		{"gc-size", required_argument, 0, LONG_OPT_GC_SIZE},
 		{"gc-count", required_argument, 0, 'G'},
-		{"wait-for-files-upto", required_argument, 0, LONG_OPT_FILE_CREATION_PATIENCE_WAIT_TIME},
```
Comment (dthain) [inline makeflow/src/makeflow.c]: On purpose?

Comment (nhazekam) [inline makeflow/src/makeflow.c]: Yes. For some reason probably due to an old code change this entry in is the struct twice. The first is on line 1404/1407.

Comment (dthain) [inline makeflow/src/makeflow.c]: Got it

**Comment by nhazekam** [comment]
Need to update test prior to merge.

**Comment by dthain** [comment]
Ready to go?

**Comment by nhazekam** [comment]
Yep, should be.

**Pull request merged**